intellij-idea/.idea/runConfigurations/msv3Server.xml

```diff
@@ -0,0 +1,10 @@
+<component name="ProjectRunConfigurationManager">
```

Comment:
I have never used this Run configuration. Do we need it in the default list?

Reply:
oh, is there also a way in intelliJ to put configs into the backseat row? that would be nice

Reply:
yes, let's chose which ones

Reply:
done